Userland/Applications/SystemMonitor/ProcessModel.h

```diff
@@ -60,6 +60,8 @@ class ProcessModel final : public GUI::Model {
     virtual int column_count(const GUI::ModelIndex&) const override;
     virtual String column_name(int column) const override;
     virtual GUI::Variant data(const GUI::ModelIndex&, GUI::ModelRole) const override;
+    virtual bool is_searchable() const override { return true; }
+    virtual Vector<GUI::ModelIndex, 1> matches(const StringView&, unsigned = MatchesFlag::AllMatching, const GUI::ModelIndex& = GUI::ModelIndex()) override;
```

Comment:
Let's use east const.
```suggestion
    virtual Vector<GUI::ModelIndex, 1> matches(StringView const&, unsigned = MatchesFlag::AllMatching, GUI::ModelIndex const& = GUI::ModelIndex()) override;
```